src/evm_state.rs

```diff
@@ -231,6 +237,10 @@ impl SubState<'_> {
 }
 
 impl EvmState for SubState<'_> {
+    fn get_ecrecover_alias(&self, address: Address) -> Option<Address> {
+        self.parent.get_ecrecover_alias(address)
```

Comment:
if parent is `impl EvmState for StateStore` will this return `None` or filter up somehow?